packages/loader/container-loader/src/deltaManager.ts

```diff
@@ -376,7 +376,7 @@ export class DeltaManager<TConnectionManager extends IConnectionManager>
 	 * we stop processing ops that results in no processing join op and thus moving to connected state)
 	 * @param event - Event to log.
 	 */
-	public logConnectionIssue(event: ITelemetryErrorEvent) {
+	public logConnectionIssue(event: ITelemetryErrorEventExt) {
```

Comment:
This only exists on `DeltaManager` the class, which isn't exported (as opposed to `IDeltaManager` the interface).